### ISSUE TITLE
plugSources does not seem correct for shaders

### ISSUE THREAD

**Issue by designker** [Bungie-net/api#522]
it returns as '2' which does not seem right as it has both a reusable plug to reset the shader and should also go get the shaders in your inventory

**Comment by vthornheart-bng** [comment]
Hmm, definitely sounds like a bug.  I'll take a look!

**Comment by vthornheart-bng** [comment]
Yep, good call.  Example of an item with this issue... anything with Shaders, but 1729074423 to point at a specific item for testing purposes.

**Comment by vthornheart-bng** [comment]
TFS 665414

**Comment by vthornheart-bng** [comment]
Found and fixed, next release we'll make sure plugSources are more consistent.  I also added some more documentation both for those of you trying to figure out plug sources via categories and for my own reference should we ever have time to loop back and make this more usable from an external perspective.
